examples/securenn/network_c.py

```diff
@@ -101,25 +76,42 @@ def build_training_graph(self, training_data) -> List[tf.Tensor]:
     Returns a list of the trained model's parameters.
     """
     # model parameters and initial values
-    wconv1 = weight_variable([self.KERNEL,
-                              self.KERNEL,
-                              self.IN_CHANNELS,
-                              self.HIDDEN_C1], 1.)
-    bconv1 = bias_variable([1, 1, self.HIDDEN_C1])
-    wconv2 = weight_variable([self.KERNEL,
-                              self.KERNEL,
-                              self.HIDDEN_C1,
-                              self.HIDDEN_C2], 1.)
-    bconv2 = bias_variable([1, 1, self.HIDDEN_C2])
-    wfc1 = weight_variable([self.HIDDEN_FC1, self.HIDDEN_FC2], 1.)
-    bfc1 = bias_variable([self.HIDDEN_FC2])
-    wfc2 = weight_variable([self.HIDDEN_FC2, self.OUT_N], 1.)
-    bfc2 = bias_variable([self.OUT_N])
-    params = [wconv1, bconv1, wconv2, bconv2, wfc1, bfc1, wfc2, bfc2]
+    model = keras.Sequential()
+    model.add(keras.layers.Conv2D(
+        self.HIDDEN_C1,
```

Comment:
Let's keep the indentation similar to network b unless for some reasons, it doesn't work.

Reply:
In the inference section this doesn't tend to work because I usually call the dimension attributes from `ModelTrainer`